### ISSUE TITLE
Changing meta description for threads?

### ISSUE THREAD

**Issue by Batemanx9** [frostschutz/MyBB-Google-SEO#64]
Hello,

I have a bit of difficulties with finding the meta description for the thread.
I would like to add the thread subject to the meta description when viewing a thread/post, where do you add this thread meta description, in what file? 
So I can modify it and add the thread subject to it, thanks!

**Comment by frostschutz** [comment]
It's in meta.php see here:

https://github.com/frostschutz/MyBB-Google-SEO/blob/e4e71f75b50054e19fcde4a3b50070c0063dc30e/inc/plugins/google_seo/meta.php#L229-L233

**Comment by Batemanx9** [comment]
Yeah, I'm also using MySEO (https://community.mybb.com/mods.php?action=view&pid=359) - Could this be the issue?
